src/Controls.js

```diff
@@ -32,17 +39,22 @@ export default class Controls extends Component {
     this.props.filterByDistance(this.state.distance)
   }
 
+  getParkName = () => {
+    this.props.filterByParkName(this.state.parkName)
+  }
+
   resetApplication = () => {
     this.props.toggleLandingScreen();
   }
 
   render() {
     return (
       <div className="controls-container">
+        <h4>Number of Trails: {this.props.trails.length}</h4>
```

Comment:
Looks like you were able to display the number of trails that populate after rendering. That is awesome.